consensus/statehash.go

```diff
@@ -18,9 +18,9 @@ func (s *State) sortedUtxoSet() (sortedOutputs []Output) {
 	for _, utxoString := range unspentOutputStrings {
 		var outputID OutputID
 		copy(outputID[:], utxoString)
-		output, err := s.output(outputID)
-		if err != nil {
-			panic(err)
+		output, exists := s.output(outputID)
+		if !exists {
+			panic("output doesn't exist?")
```

Comment:
yeah. Doing my best to remove all the sharp edges. Just needs more love is all.

Sharp edges are good in developer mode though they let you know stuff is not right.